### PR TITLE
Add improvements.

### PR DESCRIPTION
**Pull request by dewniMW**

### CODE REVIEW

File: en/docs/fragments/skip-user-consent.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
 
-1.  Access the [developer portal](insert-link). 
+1. Log in to the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) using administrator credentials (`admin:admin`).
 
-2.  Click the edit icon adjacent to the application for which you want to skip the consent step.  
+2. Navigate to the **Service Providers** tab listed under the **Identity** section and click **List**.
 
-3.  Access the **Advanced** tab. 
+3. Click **Edit** icon adjacent to the service provider for which you want to skip the consent step.
```
Comment (gomathyk) [inline en/docs/fragments/skip-user-consent.md:6]:
```suggestion
3. Click **Edit** adjacent to the service provider for which you want to skip the consent step.  
```

File: en/docs/fragments/skip-user-consent.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+1. Log in to the WSO2 Identity Server Management Console (`https://<IS_HOST>:<PORT>/carbon`) using administrator credentials (`admin:admin`).
 
-2.  Click the edit icon adjacent to the application for which you want to skip the consent step.  
+2. Navigate to the **Service Providers** tab listed under the **Identity** section and click **List**.
 
-3.  Access the **Advanced** tab. 
+3. Click **Edit** icon adjacent to the service provider for which you want to skip the consent step.  
 
-4.  Enable **Skip login consent**. 
+3. Expand **Local & Outbound Authentication Configuration** section.
```
Comment (gomathyk) [inline en/docs/fragments/skip-user-consent.md:8]:
```suggestion
3. Expand **Local & Outbound Authentication Configuration**. 
```

File: en/docs/guides/login/integrate-custom-protocol.md [hunk 3]
```diff
@@ -4,32 +4,31 @@ WSO2 Identity Server provides a way to decouple authentication functionality fro
...
-
-<a class="samplebtn_a" href="../../../quick-starts/webapp-custom-sample" rel="nofollow noopener">Try it with the sample</a>
+<!---
 
-----
+---
 
-{!fragments/deploying-sample-apps.md!}
+This guide assumes you have your own web application and a custom inbound authenticator. If you wish to try out this flow with a sample application, click the button below.--->
+<!--- a class="samplebtn_a" href="../../../quick-starts/webapp-custom-sample" rel="nofollow noopener">Try it with the sample</a --->
```
Comment (gomathyk) [inline en/docs/guides/login/integrate-custom-protocol.md:12]: Has this file been added back to the nav?

**Review by gomathyk — APPROVED** [review]

File: en/docs/guides/login/integrate-custom-protocol.md (hunk 3 above)
Comment (dewniMW) [reply en/docs/guides/login/integrate-custom-protocol.md:12]: as discussed offline, this file (quick-starts/webapp-custom-sample) is hidden from mkdocs.yml file

**Pull request merged**